CHANGELOG.md

```diff
@@ -20,7 +20,17 @@ Changes are grouped as follows
 - Separate read/write fields on data classes
 
 ## [Unreleased]
+
+### Changed
+- Sequences data retrieval now returns a SequenceData object.
+- Sequences insert takes its parameters row data first, and no longer requires columns to be passed.
+- Sequences insert now accepts tuples and raw-style data input
+- Sequences create now clears invalid fields such as 'id' in columns specification, so sequences can more easily re-use existing specifications.
+
 ### Added
+- Variety of useful helper functions for Sequence and SequenceData objects, particularly .column_ids() and .column_external_ids()
```

Comment:
Feel free to include the names of all the helper functions you made. I got some feedback on our changelog that it should be more detailed.